eng/pipelines/compliance.yml

```diff
@@ -27,11 +27,42 @@ jobs:
           targetArgument: "$(Build.SourcesDirectory)"
           result: "PoliCheck.xml"
 
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download symbols - RTM artifact"
+        inputs:
+          source: specific
+          project: 'devdiv'
+          pipeline: NuGet.Client-Official
+          artifact: "symbols - RTM"
+          path: $(Pipeline.Workspace)\symbols\RTM
+
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download symbols - NonRTM artifact"
+        inputs:
+          source: specific
+          project: 'devdiv'
+          pipeline: NuGet.Client-Official
+          artifact: "symbols - NonRTM"
+          path: $(Pipeline.Workspace)\symbols\NonRTM          
```

Comment:
The `RTM` artifacts contain a sub-set of the `NonRTM` artifacts.  Do we need to analyze both?  The only difference (should be) the version string.

Reply:
@zivkan 
I was told if possible scan both (RTM and Non-RTM) of them, but in case have choose one then RTM would be preferable option.

Reply:
Whoever gave you the advise that the RTM artifacts would be preferable over NonRTM didn't know NuGet's build scripts. Our "RTM" and "NonRTM" naming is completely misleading, so people who are making assumptions that the name accurately represents what the binaries are will be making bad assumptions. As I mentioned in my previous comment, the RTM artifacts are a subset of the NonRTM artifacts. Scanning both means we're duplicating scanning of binaries that make the same API calls, but if that's not a big deal, then it's fine.

Reply:
> The RTM artifacts contain a sub-set of the NonRTM artifacts

I didn't know that. In this case, I think we should be just analyzing `NonRTM` artifacts then.

Reply:
Ok. Removed `RTM` artifact download.